yarn.lock

```diff
@@ -13800,7 +13800,7 @@ __metadata:
   bin:
     asc: bin/asc
     asinit: bin/asinit
-  checksum: fda292f6b3b8b4a9bd4457c9143e50145f8473f3d664e48c057638a9bb1c8e890cd4de4e38e18dd4334240394621cfd9d01afee344ab049c5a7072fe855329b5
+  checksum: 8a407db6179addfaa5fcc637543a755ecc22ae8dd5e8259492305afd0d2bb73eae019d4178f79b515f75648d71dfda2c1ceb0c7421bff7817aebb2e2f5c8bfe7
```

Comment:
Any idea why this changed?

Reply:
The checksum wasn't matching on yarn install so it was failing. I think they changed the GitHub release. I forced yarn to update the checkup in lock file.

Reply:
Ok thanks for the clarification